data/progress_source.yaml

```diff
@@ -74,6 +74,15 @@ data:
     releaseName: "n/a"
     releaseDescription: "n/a"
     releaseUrl: ""
+  - label: Erlang/Elixir
```

Comment:
If you change this value to 'Erlang' it'll work - the slash in the label breaks the css class.

Reply:
Ah, *facepalm*, ok, I've pushed a change.